comictaggerlib/taggerwindow.py

```diff
@@ -1727,11 +1748,8 @@ def metadata_save() -> bool:
         ii = IssueIdentifier(ca, self.config[0], self.current_talker())
 
         # read in metadata, and parse file name if not there
-        try:
-            md = ca.read_metadata(self.load_data_style)
-        except Exception as e:
-            md = GenericMetadata()
-            logger.error("Failed to load metadata for %s: %s", ca.path, e)
+        md, success = self.overlay_ca_read_style(self.load_data_styles, ca)
```

Comment:
`success` is never used, either remove the return value from `overlay_ca_read_style` or do something with it here